pwiz_tools/Skyline/Controls/Databinding/DataboundGridControl.cs

```diff
@@ -663,149 +663,22 @@ private void heatMapContextMenuItem_Click(object sender, EventArgs e)
             ShowHeatMap();
         }
 
-        public bool ShowHeatMap()
+        public ClusterInput GetClusterInput()

```

Comment:
I tend to prefer using "Get" for things that don't create new objects, and prefixes like "Create" or "Calc" for things that do.